docs/specs/meta-clients/web3modal/api.md

```diff
@@ -1,142 +1,240 @@
+
+Fetches image for known tokens using the token's ticker symbol.
+
+#### Route Parameters
+
+- `token_symbol`: The token's ticker symbol (e.g., `ETH`, `USDC`, `WBTC`)
+- `size`: The predefined image size, described as `{width}x{height}` (see [Known Image Sizes](#known-image-sizes))
+
+#### Example API Requests
+
+- Ethereum: `GET /v2/asset-image/token/56x56/ETH`
+- USD Coin: `GET /v2/asset-image/token/56x56/USDC`
+- Wrapped Bitcoin: `GET /v2/asset-image/token/56x56/WBTC`
+
+#### Response
+
+- A binary image file (Blob)
+
+### GET `/v2/asset-image/currency/{size}/{currency_symbol}`
+
+Fetches image for known fiat currencies using the currency's symbol.
+
+#### Route Parameters
+
+- `currency_symbol`: The currency's symbol (e.g., `USD`, `EUR`, `JPY`)
+- `size`: The predefined image size, described as `{width}x{height}` (see [Known Image Sizes](#known-image-sizes))
+
+#### Example API Requests
+
+- US Dollar: `GET /v2/asset-image/currency/56x56/USD`
+- Euro: `GET /v2/asset-image/currency/56x56/EUR`
+- Japanese Yen: `GET /v2/asset-image/currency/56x56/JPY`
+
+#### Response
+
+- A binary image file (Blob)
+
+### `GET /v2/analytics-config`
+
+Fetches the analytics configuration for the project based on the mandatory `projectId=` query parameter.
+
+#### Response
 
 ```typescript
-// Maps EIP155 chain IDs -> `asset_id`
-const EIP155NetworkImageIds = {
-  // Ethereum
-  1: "692ed6ba-e569-459a-556a-776476829e00",
-  // Arbitrum
-  42161: "600a9a04-c1b9-42ca-6785-9b4b6ff85200",
-  // Avalanche
-  43114: "30c46e53-e989-45fb-4549-be3bd4eb3b00",
-  // Binance Smart Chain
-  56: "93564157-2e8e-4ce7-81df-b264dbee9b00",
-  // Fantom
-  250: "06b26297-fe0c-4733-5d6b-ffa5498aac00",
-  // Optimism
-  10: "ab9c186a-c52f-464b-2906-ca59d760a400",
-  // Polygon
-  137: "41d04d42-da3b-4453-8506-668cc0727900",
-  // Gnosis
-  100: "02b53f6a-e3d4-479e-1cb4-21178987d100",
-  // Evmos
-  9001: "f926ff41-260d-4028-635e-91913fc28e00",
-  // ZkSync
-  324: "b310f07f-4ef7-49f3-7073-2a0a39685800",
-  // Filecoin
-  314: "5a73b3dd-af74-424e-cae0-0de859ee9400",
-  // Iotx
-  4689: "34e68754-e536-40da-c153-6ef2e7188a00",
-  // Metis,
-  1088: "3897a66d-40b9-4833-162f-a2c90531c900",
-  // Moonbeam
-  1284: "161038da-44ae-4ec7-1208-0ea569454b00",
-  // Moonriver
-  1285: "f1d73bb6-5450-4e18-38f7-fb6484264a00",
-  // Zora
-  7777777: "845c60df-d429-4991-e687-91ae45791600",
-  // Celo
-  42220: "ab781bbc-ccc6-418d-d32d-789b15da1f00",
-  // Base
-  8453: "7289c336-3981-4081-c5f4-efc26ac64a00",
-  // Aurora
-  1313161554: "3ff73439-a619-4894-9262-4470c773a100",
-};
-
-// Maps connector name -> `asset_id`
-const ConnectorImageIds = {
-  coinbaseWallet: "0c2840c3-5b04-4c44-9661-fbd4b49e1800",
-  safe: "461db637-8616-43ce-035a-d89b8a1d5800",
-  ledger: "54a1aa77-d202-4f8d-0fb2-5d2bb6db0300",
-  walletConnect: "ef1a1fcf-7fe8-4d69-bd6d-fda1345b4400",
-  injected: "07ba87ed-43aa-4adf-4540-9e6a2b9cae00",
-};
+{
+  isAnalyticsEnabled: boolean,
+}
 ```
+
+## Known Image Sizes
```

Comment:
@skibitsky I remember you mentioning during the review call that you'd need power-of-2-based image dimensions, are there specific ones we should already define here?

Reply:
I ran some tests but didn't see any performance improvements for images downloaded in realtime. As long as the images are square we should be fine! 👍